content/en/docs/instrumentation/swift/automatic.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Automatic Instrumentation
+linkTitle: Automatic
+weight: 5
+description: Automatic instrumentation for opentelemetry-swift
+---
+
+OpenTelemetry-Swift provides several instrumentation libraries.
+
+## `SDKResourceExtension`
+
+`SDKResourceExtension` provides details about the device as a Resource.
+
+### Usage
+
+Use `DefaultResource.get()` to generate an all-in-one resource object. This
+resource can be added to a `TracerProvider` or `MetricProvider`.
+
+```swift
+OpenTelemetry.registerMeterProvider(meterProvider: MeterProviderBuilder()
+            .with(resource: DefaultResource.get())
+            .build())
+```
+
+### Details
+
+`SDKResourceExtension` provides attributes in a resource object with details
+about the iOS device, OS details, and application details. It applies these
+values to the appropriate
+[semantic attributes](/docs/reference/specification/resource/semantic_conventions/#semantic-attributes-with-sdk-provided-default-value).
+
+#### Application Info
+
+| Attribute           | Value example                 | Description                                                                                              |
+| ------------------- | ----------------------------- | -------------------------------------------------------------------------------------------------------- |
+| `service.name`      | `MyApplication`               | `CFBundleName`; The application name defined in the App's info.plist.                                    |
+| `service.version`   | `1.0 (1234)`                  | `CFBundleShortVersion` & (`CFBundleVersion`); The application version as defined in the App's info.plist |
+| `service.namespace` | `com.myCompany.myApplicaiton` | `CFBundleIdentifier`                                                                                     |
```

Comment:
this probably doesn't belong here but in the swift repo: Shouldn't namespace be `com.myCompany` with out the service.name?

Reply:
This is a good point. I'll leave it as is here, since we are documenting the existing library, but I'll bring it up in the sig meeting